organization/data-processing-tools.md

```diff
@@ -31,3 +31,11 @@ We use Tentoo for human resources management, including processing salaries and
 Tentoo's [General Terms and Conditions](https://www.tentoo.nl/images/downloads/Generaltermsandconditions_TentooPartners.pdf) say that Tentoo doesn't process data outside the European Economic Area (EEA):
 
 > 6.1 Tentoo will only process the Personal Data in the European Economic Area (EEA), and will ensure that no processing actions at all take place outside the EEA, unless Tentoo has obtained the Client’s explicit, written consent to this.
+
+## Odoo
+
+We use Odoo Community for customer relationship management (CRM) and project management for our codebases and membership development. Opus VL is our hosting provider for Odoo Community.
+
+Opus VL's describes its [data processing activities in their privacy policy](https://opusvl.com/privacy-policy).
```

Comment:
```suggestion
Opus VL's data processing activities are described in their [privacy policy](https://opusvl.com/privacy-policy).
```

Reply:
More urgent concern: this privacy policy is for the opusvl.com website, not for hosting provided by OpusVL the company.

Reply:
So this needs to be resolved first @demkodo